akka-http-testkit/src/main/scala/akka/http/javadsl/testkit/RouteTest.scala

```diff
@@ -34,7 +34,8 @@ abstract class RouteTest extends AllDirectives with WSTestRequestBuilding {
   implicit def materializer: Materializer
   implicit def executionContext: ExecutionContextExecutor = system.dispatcher
 
-  protected def awaitDuration: FiniteDuration = 3.seconds.dilated
+  protected def defaultAwaitDuration = 3.seconds
```

Comment:
Introduced `defaultAwaitDuration` so it's simpler to override the `awaitDuration` from java code (where invoking `dilated` is not available without too much machinery)